airgun/views/computeresource.py

```diff
@@ -241,15 +241,29 @@ class containers(SatTab, SearchableViewMixin):
         )
 
 
+class ComputeResourceLibvirtProfileNetworkItem(GenericRemovableWidgetItem):
+    """Libvirt Compute Resource Profile "Network interface" item widget"""
+    network_type = FilteredDropdown(id="type")
+    network = FilteredDropdown(id="bridge")
+    nic_type = FilteredDropdown(id="model")
+
+
+class ComputeResourceLibvirtProfileStorageItem(GenericRemovableWidgetItem):
+    """Libvirt Compute Resource profile "Storage" item widget"""
+    storage_pool = FilteredDropdown(id="pool_name")
+    size = TextInput(locator=".//input[contains(@id, 'capacity')]")
+    storage_type = FilteredDropdown(id="format_type")
+
```

Comment:
can we include the `Allocation (GB)`  too  ..
what do you think ?

Reply:
@vijay8451 Let's me move that out of the scope of that PR as it is probably require separate widget for it:
![image](https://user-images.githubusercontent.com/10895065/58480292-c1389600-8162-11e9-992e-d01a267acd26.png)

Reply:
@oshtaier yeah it looks more a new widget but I have seen same used by multiple Compute resource hence worth to have that. 
as new issue #330 opened , I am fine with it :)